events.json

```diff
@@ -94,10 +119,6 @@
             }
         ],
         "achievements": [
-            {
```

Comment:
Why was this achievement removed?

Reply:
Oh, how could this happen? I didn't consciously remove that. I look at it and correct it, thank you very much!